src/components/Dropdown/Dropdown.vue

```diff
@@ -44,43 +44,37 @@
     >
       <div
         :class="[
-          'mr-8 truncate',
-          {'text-xs': small},
-          {'text-gray-100': disabled},
-          {'text-gray-900 !font-normal' : open || activeIndex > -1},
-          {'text-gray-500' : activeIndex < 0},
-          {'text-error': error}
+          'mr-8 truncate type-small-500',
+          value ? 'text-gray-800' : 'text-gray-500',
+          { '!text-gray-300': disabled },
+          { 'text-gray-800' : open || activeIndex > -1 },
+          { 'text-gray-500' : activeIndex < 0 },
+          { 'text-green-600': success },
+          { 'text-red-600': error }
         ]"
       >
         {{ value || placeholder }}
       </div>
       <chevron-down
-        v-if="!open"
         size="s"
         :class="[
-          'absolute right-2.5',
-          error ? 'text-error' : 'text-gray-900'
+          'absolute right-2.5 text-gray-500 transition-all duration-100',
```

Comment:
should we have `text-gray-500` only when `!success && !error && !disabled`?

Reply:
same question above

Reply:
this the Chevron and it seems to be following text color on the designs - I can double check this w Michael

Reply:
update: added this as suggested! thnx